src/ape_plugins/_cli.py

```diff
@@ -122,12 +122,31 @@ def _list(cli_ctx, display_all):
             click.echo()
 
 
+def upgrade_option(help=""):
+    """
+    A ``click.option`` for upgrade plugin confirmation (``--upgrade``).
+
+    Args:
+        help (str): CLI option help text. Defaults to ``""``.
+    """
+
+    return click.option(
+        "-U",
+        "--upgrade",
+        "upgrade_option",
```

Comment:
you don't need to put `option` in the name.

every parameter to the commands are options.

just calling it `upgrade` is more expected, which is what it will default to
```suggestion
```

Reply:
I know it seems weird, but this is just a `click`-ism